lib/nerves_hub/logger_formatter.ex

```diff
@@ -0,0 +1,7 @@
+defmodule NervesHub.LoggerFormatter do
+  def format(level, message, timestamp, metadata) do
+    pattern = Logger.Formatter.compile("$time [$level] $metadata$message\n")
+    metadata = Keyword.drop(metadata, [:line, :file, :domain, :application])
+    Logger.Formatter.format(pattern, level, message, timestamp, metadata)
+  end
```

Comment:
```suggestion
  @pattern Logger.Formatter.compile("$time [$level] $metadata$message\n")
  def format(level, message, timestamp, metadata) do
    metadata = Keyword.drop(metadata, [:line, :file, :domain, :application])
    Logger.Formatter.format(@pattern, level, message, timestamp, metadata)
  end
```

Just a small optimization to precompile the format rather than compile it on each log/formatting event